v2/thirdpartyemailpassword/migration/bulk-user-import-schema-examples.mdx

```diff
@@ -0,0 +1,200 @@
+---
+id: bulk-user-import-schema-examples
+title: Bulk User Import Schema and Examples
+hide_title: true
+---
+
+# Bulk User Import Schema and Examples
+
+
+The bulk import API expects the payload in the following format. 
+
+```json
+{
+  "users": {
+    "type": "array",
+    "items": {
```

Comment:
Is it possible to render this in some way? it's way too confusing. People who don't know about json schema, won't even understand this.